MC-37830: [Documentation] New Products List (#806)
magento/merchdocs: src/cms/widget-new-products-list.md

## Step 1: Set the date range for each product

{:.bs-callout-info}

> For Magento Commerce customers, see [Scheduling an Update]({% link cms/content-staging-scheduled-update.md %}) and then continue to the Step 2 of this manual.

Reviewer: ```suggestion
For Magento Commerce customers, see [Scheduling an Update]({% link cms/content-staging-scheduled-update.md %}) and then continue to the Step 2 on this page.
```
